cmd/ctr/commands/containers/containers.go

```diff
@@ -26,11 +28,14 @@ import (
 	"github.com/containerd/containerd"
 	"github.com/containerd/containerd/cio"
 	"github.com/containerd/containerd/cmd/ctr/commands"
-	"github.com/containerd/containerd/cmd/ctr/commands/run"
 	"github.com/containerd/containerd/containers"
 	"github.com/containerd/containerd/errdefs"
+	clabels "github.com/containerd/containerd/labels"
 	"github.com/containerd/containerd/log"
+	"github.com/containerd/containerd/oci"
 	"github.com/containerd/typeurl"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sirupsen/logrus"
```

Comment:
question : Why is logrus used directly instead of `"github.com/containerd/containerd/log"`. I can see that some packages used `logrus`, and some uses `containerd/log`.

Reply:
This seems like a historical reason, but for `cmd/ctr`, I think we should use the unified containerd/log instead of the original logrus.

But that's irrelevant for this pr, we can unify them in the new pr.